setup.py

```diff
@@ -67,6 +67,8 @@ def run_tests(self):
             "scriptworker = scriptworker.worker:main",
             "create_initial_gpg_homedirs = scriptworker.gpg:create_initial_gpg_homedirs",
             "rebuild_gpg_homedirs = scriptworker.gpg:rebuild_gpg_homedirs",
+            "nagios_check_pending_tasks = scriptworker.monitoring.nagios_pending_tasks:query_pending_task_count",
+            "nagios_check_file_ages = scriptworker.monitoring.nagios_file_age_check:run_file_age_checks",
```

Comment:
A side effect of this will be a reduction in overall test coverage because there are no tests in this PR.

We can fix this in a couple ways:
1. add tests.  this is extra work, but having tests seems good... let me know if you need a hand with this.
2. these scripts import from scriptworker, but we can still potentially install them via puppet http://hg.mozilla.org/build/puppet/file/tip/modules/scriptworker/files , bypassing the package coverage requirement.